### PR TITLE
GCI-142: fixed minor bugs in js

### PR DESCRIPTION
**Pull request by ShobhitRathi**
## Description of what I changed
1. Add curly braces around the nested statement(s) in this "if" block **&**
2. Replace "==" with "==="

**Issue** [link](https://sonar.openmrs.org/project/issues?id=org.openmrs%3Aopenmrs&open=AWCS0QlUvsycEPA9_R41&resolved=false&rules=javascript%3ACurlyBraces)

## Issue I worked on
_OpenMRS_ [Issue page](https://issues.openmrs.org/browse/GCI-142)
_Google Code-in 2019_ [task](https://codein.withgoogle.com/dashboard/task-instances/5161208928272384/)

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/jquery-ui-timepicker-addon.js [hunk 1]
```diff
@@ -182,29 +182,34 @@ $.extend(Timepicker.prototype, {
...
 				.css({ cursor: 'pointer' })
 				.focus(function(){ $input.trigger("focus"); });
+		}
 		
 		if(tp_inst._defaults.minDate==0 || tp_inst._defaults.minDateTime==0)
 		{
 			tp_inst._defaults.minDate=new Date();
 		}
-		if(tp_inst._defaults.maxDate==0 || tp_inst._defaults.maxDateTime==0)
+		if(tp_inst._defaults.maxDate===0 || tp_inst._defaults.maxDateTime===0)
```
Comment (suthagar23) [inline webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/jquery-ui-timepicker-addon.js]: Format this line with spacing next to equal sign

File: webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/jquery-ui-timepicker-addon.js [hunk 2]
```diff
@@ -182,29 +182,34 @@ $.extend(Timepicker.prototype, {
...
 		{
 			tp_inst._defaults.maxDate=new Date();
 		}
 		
 		// datepicker needs minDate/maxDate, timepicker needs minDateTime/maxDateTime..
-		if(tp_inst._defaults.minDate !== undefined && tp_inst._defaults.minDate instanceof Date)
+		if(tp_inst._defaults.minDate !== undefined && tp_inst._defaults.minDate instanceof Date) {
```
Comment (suthagar23) [inline webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/jquery-ui-timepicker-addon.js]: What do you think about,
```suggestion
		if(tp_inst._defaults.minDate && tp_inst._defaults.minDate instanceof Date) {
```

Comment (ShobhitRathi) [reply webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/jquery-ui-timepicker-addon.js]: Hi, don't know how this happened ill fix it just now thanks for telling me 😄

**Comment by PrathameshMutkure** [comment]
Hii @ShobhitRathi, have a look at [OpenMRS Pull Request Tips](https://wiki.openmrs.org/display/docs/Pull+Request+Tips), make sure to squash all commits into 1

Here's an [artical](https://github.com/todotxt/todo.txt-android/wiki/squash-all-commits-related-to-a-single-issue-into-a-single-commit) to help you :grinning:

**Comment by ShobhitRathi** [comment]
Hi @PrathamushMutkure, I didn't knew this before so If mentor doesn't mind I hope thats good.

File: webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/jquery-ui-timepicker-addon.js [hunk 3]
```diff
@@ -182,29 +182,34 @@ $.extend(Timepicker.prototype, {
...
 			tp_inst._defaults.maxDate=new Date();
 		}
 		
 		// datepicker needs minDate/maxDate, timepicker needs minDateTime/maxDateTime..
-		if(tp_inst._defaults.minDate !== undefined && tp_inst._defaults.minDate instanceof Date)
+		if(tp_inst._defaults.minDate && tp_inst._defaults.minDate instanceof Date) {
 			tp_inst._defaults.minDateTime = new Date(tp_inst._defaults.minDate.getTime());
-		if(tp_inst._defaults.minDateTime !== undefined && tp_inst._defaults.minDateTime instanceof Date)
+		}
+		if(tp_inst._defaults.minDateTime !== undefined && tp_inst._defaults.minDateTime instanceof Date) {
```
Comment (suthagar23) [inline webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/jquery-ui-timepicker-addon.js]: Did you forgot to change this and other lines below this?

Comment (ShobhitRathi) [reply webapp/src/main/webapp/WEB-INF/view/scripts/jquery-ui/js/jquery-ui-timepicker-addon.js]: Yes, check out my new [PR](https://github.com/openmrs/openmrs-core/pull/3089)

**Pull request closed without merging**